src/managers/NotificationsManager/NotificationsManager.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { createNotification } from './api';
+
+<Meta title="managers/NotificationsManager/Docs" />
+
+# NotificationsManager
+
+This feature provides an api to display toast notifications programatically
+
+## Usage
+
+First, you need to wrap your application with the `NotificationsProvider` component.
+
+```jsx
+import { NotificationsManager } from '@securityscorecard/design-system';
```

Comment:
In the sentence above you are using `NotificationsProvider` but here you are using `NotificationsManager`.
I'd suggest to remain consistent with the banners management and use `NotificationsProvider` instead of Manager